discovery-provider/src/utils/db_session.py

```diff
@@ -1,70 +1,16 @@
-import ast
-from contextlib import contextmanager
-from sqlalchemy import create_engine
-from sqlalchemy.orm import sessionmaker
-from flask import current_app, g
-
-
-session_manager = None
-
+from flask import current_app
 
 def get_db():
     """Connect to the configured database. The connection
     is unique for each request and will be reused if this is called
     again.
     """
-    if "db" not in g:
-        g.db = SessionManager(
-            current_app.config["db"]["url"],
-            ast.literal_eval(current_app.config["db"]["engine_args_literal"]),
-        )
-
-    return g.db
+    return current_app.db_session_manager
```

Comment:
do we actually need get_db anywhere? looks like its only ever used in tests, wondering if those usages can be replaced with get_db_read_replica - non blocking for this though

Reply:
yeah probably, should take a closer look into that :)